src/ui/components/atoms/logo-footer/LogoFooter.module.scss

```diff
@@ -11,11 +11,11 @@
   height: 4.5rem;
   width: 11.875rem;
 
-  &--primary {
+  &.primary {
```

Comment:
Why are you change the css ?

Reply:
Por esto, ya que el noción dice que tomemos de referencia el ticket SEM-122
![image](https://github.com/Ditmar/OpenScience/assets/113062275/c751a17b-0d82-4b5e-a723-b370c8e34fa1)
![image](https://github.com/Ditmar/OpenScience/assets/113062275/aa91284c-0531-47e7-bba6-0a7931ae10ff)